modules/syslog.lua

```diff
@@ -13,10 +13,16 @@ local tonumber = tonumber
 local error = error
 local type = type
 local ipairs = ipairs
+local rawset = rawset
```

Comment:
This is how I currently use it in Heka: https://github.com/jedisct1/heka/commit/29430196298b70575eb2f35323ccbf57e1a7d111